examples/gzip/service.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from flask.helpers import send_from_directory
+
+app = Flask(__name__)
+
+
+@app.route('/plain')
+def get_plain_file():
+    return send_from_directory("data", "file")
```

Comment:
can we rename this to eg `file.txt` - it will make it a bit clearer i think

Reply:
OK, unify as `file.xxx`.